components/prompt/StartPrompt.tsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from "react";
+
+import styles from "styles/components/Start.module.css";
+
+type props = {
+  startAnimation: () => void;
+};
+
+export const StartPrompt = ({ startAnimation }: props) => {
+  const [animate, setAnimate] = useState(false);
```

Comment:
Why did you create another state here, can't you just pass the animate state as prop along with the setState? With that you can remove onAnimationStart and just use onClick to setAnimate true from the parent.

Reply:
The animation chain starts from the play button on the prompt and not the parent. That's why I passed a callback to trigger the parent's animation. I guess I can also call startAnimation in the onClick together with setAnimate and remove onAnimationStart.